src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`}
```

Comment:
please create a variable for this check as I wrote previously

`const partner = sex === 'f' ? 'husband' : 'wife';`

also you can do this ternary check `{isMarried ? (` inside one `p` tag, so you will not write it twice and write less code